wbooks/src/app/components/Book/style.ts

```diff
@@ -0,0 +1,37 @@
+import { StyleSheet } from 'react-native';
+import { white, fontGray } from '@constants/colors';
+
+const style = StyleSheet.create({
+  bookContainer: {
+    display: 'flex',
+    backgroundColor: white,
+    flexDirection: 'row',
+    borderRadius: 8,
+    paddingVertical: 15,
+    paddingHorizontal: 30,
+    height: 90,
+    alignItems: 'center',
+    marginVertical: 5
+  },
+  bookInfoContainer: {
+    flex: 1,
+    flexDirection: 'column',
+    marginLeft: 15
```

Comment:
if you analyze paddingVertical, paddingHorizontal and marginLeft can share a common value, you could create a default space